include/dylib.hpp

```diff
@@ -31,16 +31,25 @@
 class dylib {
 public:
 #if defined(_WIN32) || defined(_WIN64)
+    static constexpr const char *prefix = "";
     static constexpr const char *extension = ".dll";
     using native_handle_type = HINSTANCE;
 #elif defined(__APPLE__)
+    static constexpr const char *prefix = "lib";
     static constexpr const char *extension = ".dylib";
     using native_handle_type = void *;
 #else
+    static constexpr const char *prefix = "lib";
     static constexpr const char *extension = ".so";
     using native_handle_type = void *;
 #endif
 
+    static constexpr auto os_library_paths = "";
```

Comment:
1. You only define this once.
2. You say "paths" plural, but it's a single string.
3. The value doesn't seem to make sense.